### ISSUE TITLE
Reproducibility of training procedure for hand pose model

### ISSUE THREAD

**Issue by philipturner** [google/mediapipe#4218]
Can you provide the data set used to train the [hand pose model](https://arxiv.org/pdf/2006.10214.pdf)? At the very least, could you provide the rendering software used for generating artifical images? I'd like to modify the model to accept LiDAR depth data, and want to retrain the model with depth as an input texture. I'd also like to retrain to utilize the 4:3 aspect ratio from ARKit instead of 1:1.

How much compute power was required to train the original model? How many GPU-hours and on what hardware? To summarize, I am asking for:
- Real-world annotated data set used to train
- Software used to generate artificial hands
- Learning rate, batch size, and number of epochs to train the model

> For questions on how to work with MediaPipe, or support for problems that are not verified bugs in MediaPipe, please go to [StackOverflow](https://stackoverflow.com/questions/tagged/mediapipe) and [Slack](https://mediapipe.page.link/joinslack) communities.

I am unable to join the slack with my `@gmail.com` email.

**Comment by baronha** [comment]
> Can you provide the data set used to train the [hand pose model](https://arxiv.org/pdf/2006.10214.pdf)? At the very least, could you provide the rendering software used for generating artifical images? I'd like to modify the model to accept LiDAR depth data, and want to retrain the model with depth as an input texture. I'd also like to retrain to utilize the 4:3 aspect ratio from ARKit instead of 1:1.
> 
> How much compute power was required to train the original model? How many GPU-hours and on what hardware? To summarize, I am asking for:
> 
> * Real-world annotated data set used to train
> * Software used to generate artificial hands
> * Learning rate, batch size, and number of epochs to train the model
> 
> > For questions on how to work with MediaPipe, or support for problems that are not verified bugs in MediaPipe, please go to [StackOverflow](https://stackoverflow.com/questions/tagged/mediapipe) and [Slack](https://mediapipe.page.link/joinslack) communities.
> 
> I am unable to join the slack with my `@gmail.com` email.

When it's done, let everyone see the results. I'm looking forward to your product

**Comment by philipturner** [comment]
> When it's done, let everyone see the results. I'm looking forward to your product

I already did make 3D hand reconstructor using [ARHeadsetKit](https://github.com/philipturner/ARHeadsetKit). It was entirely heuristic based, building off Apple's 2D Vision model. I'm considering rewriting ARHeadsetKit from scratch to fully utilize LiDAR. MediaPipe could be much more power efficient and higher quality than my current hand tracker.

**Comment by kuaashish** [comment]
@philipturner,
Unfortunately, Sharing the details about to real-world annotated data set used to train, software used to generate artificial hands, learning rate, batch size, number of epochs to train the model are out of scope and such details are confidential. You can see [here](https://drive.google.com/file/d/1-rmIgTfuCbBPW_IFHkh3f0-U_lnGrWpg/preview) the model training details, Model Type, Model Architecture, Performance, Environment models are trained all available solution in mediapipe. However, Another issue about the joining the slack community. You can join through [this](https://mediapipe.slack.com/join/shared_invite/zt-1jqmrzg34-waJBaqdLt7_228BUOgxPwg#/shared-invite/email) invitation.

**Comment by philipturner** [comment]
> learning rate, batch size, number of epochs to train the model are out of scope and such details are confidential

Thanks for your response. I have a lot of skills in 3D rendering. If the rendering software is not going to be open-sourced for scientific reproducibility, I can just bypass this restriction: write new software myself. The manually labeled images were mostly helpful for the palm detection layer, which I do not need to retrain.

I just wonder how I’m going to train the localization layer without images from real-life. Is it too much to ask for an order-of-magnitude estimate of the compute required to train?

**Comment by philipturner** [comment]
My budget is up to one GPU-month with a machine 1/10 as good as an A100.

**Comment by philipturner** [comment]
Closing as it seems unlikely I will receive any more information.
